model/map/src/main/java/org/keycloak/models/map/realm/entity/MapAuthenticatorConfigEntity.java

```diff
@@ -45,7 +45,9 @@ static AuthenticatorConfigModel toModel(MapAuthenticatorConfigEntity entity) {
         AuthenticatorConfigModel model = new AuthenticatorConfigModel();
         model.setId(entity.getId());
         model.setAlias(entity.getAlias());
-        model.setConfig(entity.getConfig());
+        Map<String, String> config = new HashMap<>();
+        if (entity.getConfig() != null) config.putAll(entity.getConfig());
+        model.setConfig(config);
```

Comment:
Ok, without this change this [test fails](https://github.com/keycloak/keycloak/blob/main/testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/model/OwnerReplacementTest.java#L293) because the config is "live", meaning that changes to the config, like adding or removing properties end up being persisted.

This change makes this entity consistent with other entities that also have a config map. (See the [MapComponentEntity](https://github.com/keycloak/keycloak/blob/main/model/map/src/main/java/org/keycloak/models/map/realm/entity/MapComponentEntity.java#L55))

Reply:
I see, thank you!